docs/source/basic_patterns.rst

```diff
@@ -176,3 +176,29 @@ subscribed to topic ``a`` only and *Dave* subscribed to topic ``b`` only.
 
 Note how we can specify different handlers for different topics when
 subscribing agents.
+
+.. index:: more_on_filtering
+.. _more_on_filtering:
+
+More on filtering
+-----------------
+
+We have seen how to filter messages on agents at the subscriber side of the
+communication.
+
+It is also remarkable that we can easily modify the subscriptions at runtime
+easily.
```

Comment:
Maybe we can simplify this two paragraphs and say it like: "We can also very easily modify the subscriptions at run-time." (says the same with less words?).